modules/meeting/app/helpers/meetings_helper.rb

```diff
@@ -72,4 +72,14 @@ def render_journal_details(journal, header_label = :label_updated_time_by, _mode
 
     content_tag('div', "#{header}#{details}".html_safe, id: "change-#{journal.id}", class: 'journal')
   end
+
+  def global_create_context?
+    request.path == new_meeting_path
+  end
+
+  def options_for_project_selection
```

Comment:
This can now be removed since the functionality is now provided by the filters passed into the autocompleter.